### DISCUSSION TITLE
Multi Day PPP - Sinex Bias concatenation

### DISCUSSION THREAD

**Discussion by tagliagiulio** [groops-devs/groops#266] [General]
Dear GROOPS developers,

I am trying to use GROOPS to adjust the observation of multiple days of one stattion in PPP mode using Ephemerids and bias from an IGS analysis center.
To do that, I first covert the file in IGS format from one analysis center to GROOPS attitude, clock, observation, and orbit file and then concatenate them. (Is it the right way?).
However, I did not find a way to concatenate the signalBias files. In the GnssSinexBias2SignalBias description, I read, "Time-variable biases are not supported at the moment." Does this mean that multi-day PPP is not possible at the moment? or am I doing it the wrong way?

Many thanks and best regards

Giulio Tagliaferro

**Comment by tmayerguerr** [comment]
Hi Giulio,
You are right, GROOPS expects constant biases for the calculation interval. So I don't see how you can perform your idea in GROOPS. Alternative idea: Maybe you can process/write daily normal equations and then stack them?
best regards
Torsten

**Reply by tagliagiulio** [reply]
Hi Torsten,

Thank you for your answer. I like the idea of stacking the normal equations. I will try it.

Best

Giulio
